Description: Include configuration from build/config/linux/libva,
otherwise header files are not found.
Author: Olivier Tilloy <olivier.tilloy@canonical.com>
Uploader: Nathan Pratta Teodosio <nathan.teodosio@canonical.com>
Bug-Ubuntu: https://code.launchpad.net/~nteodosio/chromium-browser/+git/chromium-browser/+merge/428038/comments/1134676
Forwarded: https://chromium-review.googlesource.com/c/chromium/src/+/3866872

--- a/media/gpu/vaapi/BUILD.gn
+++ b/media/gpu/vaapi/BUILD.gn
@@ -30,6 +30,7 @@
     "//base",
     "//media/gpu:buildflags",
   ]
+  configs = [ "//build/config/linux/libva" ]
 }

 source_set("vaapi") {
--- a/media/gpu/BUILD.gn
+++ b/media/gpu/BUILD.gn
@@ -185,6 +185,7 @@
   }

   if (use_vaapi) {
+    configs += [ "//build/config/linux/libva" ]
     public_deps += [
       "//media/gpu/vaapi",
       "//media/gpu/vaapi:common",
--- a/media/gpu/sandbox/BUILD.gn
+++ b/media/gpu/sandbox/BUILD.gn
@@ -18,6 +18,7 @@
     "//sandbox/policy:policy",
   ]
   if (use_vaapi) {
+    configs += [ "//build/config/linux/libva" ]
     deps += [ "//media/gpu/vaapi:common" ]
   }
 }
--- a/media/gpu/chromeos/BUILD.gn
+++ b/media/gpu/chromeos/BUILD.gn
@@ -35,6 +35,7 @@
   ]

   if (use_vaapi) {
+    configs += [ "//build/config/linux/libva" ]
     deps += [
       "//media/gpu/vaapi",
       "//media/gpu/vaapi:common",
@@ -134,6 +135,7 @@
     "fourcc.cc",
     "fourcc.h",
   ]
+  configs += [ "//build/config/linux/libva" ]

   deps = [
     "//base",
--- a/content/gpu/BUILD.gn
+++ b/content/gpu/BUILD.gn
@@ -43,7 +43,10 @@
     "in_process_gpu_thread.h",
   ]

-  configs += [ "//content:content_implementation" ]
+  configs += [
+    "//content:content_implementation",
+    "//build/config/linux/libva",
+  ]

   deps = [
     "//base",
